Duality/Resources/Font.cs

```diff
@@ -382,12 +387,15 @@ public void UpdateKerningData()
 
 				for (int i = 0; i < this.glyphs.Length; ++i)
 				{
-					PixelData glyphTemp = this.GetGlyphBitmap(this.glyphs[i].Glyph);
+					PixelData glyphTemp_ = this.GetGlyphBitmap(this.glyphs[i].Glyph);
+					PixelData glyphTemp = new PixelData(this.glyphs[i].Width, this.glyphs[i].Height + this.glyphs[i].OffsetY);
+					glyphTemp_.DrawOnto(glyphTemp, BlendMode.Solid, 0, this.glyphs[i].OffsetY);

```

Comment:
I thought so too but did not understand the algorithm used and wanted something that works - also I thought since it's only really used when generating the font (generally in the importer) that it wasn't so risky. I'll have a look at the algorithm to see if I can update.